docs/integrations/airflow.md

```diff
@@ -9,30 +9,38 @@ redirect_from: ../using/airflow.html
 ---
 
 # Using lakeFS with Airflow
+
 [Apache Airflow](https://airflow.apache.org/) is a platform to programmatically author, schedule and monitor workflows.
 
-There are some aspects we will need to handle in order to run Airflow with lakeFS:
+There are several steps needed to run Airflow with lakeFS.
+
+## Create a lakeFS connection on Airflow
 
-## Creating the lakeFS connection
-For authenticating to the lakeFS server, you need to create a new [Airflow Connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html)
-of type HTTP and pass it to your DAG. You can do that using the Airflow UI or the cli.
-Here’s an example Airflow command that does just that:
+To access the lakeFS server and authenticate with it, create a new [Airflow
+Connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html)
+of type HTTP and pass it to your DAG. You can do that using the Airflow UI
```

Comment:
what about: "pass it to your airflow DAG" ?

Reply:
Good point, I wrote it backwards.  Going with "add it to your DAG": the command _literally_ says "airflow connection(s) add".